package_scanner.nim

```diff
@@ -41,24 +42,44 @@ const
 
   VCS_TYPES = @["git", "hg"]
 
+proc canFetchNimbleRepository(name: string, urlJson: JsonNode): bool =
+  # The fetch is a lie!
+  # TODO: Make this check the actual repo url and check if there is a
+  #       nimble file in it
+  result = true
+  var url: string
+
+  if not urlJson.isNil:
+    url = urlJson.str
+
+    try:
+      discard getContent(url, timeout=1000)
```

Comment:
As mentioned, we were not actually using the git:// URLs before and we are not now either. Notice how it is the "web" URL that is used and not the "url" URL that is used. :wink: 

The TODO is about fixing that but fixing that is hard. See my reply below.